packages/flame/test/extensions/aabb_test.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:math';
+
+import 'package:flame/extensions.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('aabb extensions tests', () {
```

Comment:
I don't remember what we usually name the outer groups, it should be in our docs here:
https://docs.flame-engine.org/1.4.0/development/testing_guide.html
If I remember correctly it should be the name of the class, or in this case the extension.